spec/features/results_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature "shows results" do
+  scenario "shows results" do
+    visit('/play')
+    # select Rock
```

Comment:
Amazing start here. This page has some hints about how to select from radio buttons: https://devhints.io/capybara